cmd/ctr/commands/images/images.go

```diff
@@ -355,3 +356,64 @@ var removeCommand = cli.Command{
 		return exitErr
 	},
 }
+
+var pruneCommand = cli.Command{
+	Name:        "prune",
+	Usage:       "remove unused images",
+	ArgsUsage:   "[flags] [<filter>, ...]",
+	Description: "remove one or more unused images",
+	Action: func(context *cli.Context) error {
+		client, ctx, cancel, err := commands.NewClient(context)
+		if err != nil {
+			return err
+		}
+
+		defer cancel()
+		var (
+			exitErr        error
+			filters        = context.Args()
+			imageStore     = client.ImageService()
+			containerStore = client.ContainerService()
+		)
+
+		imageList, err := imageStore.List(ctx, filters...)
+		if err != nil {
+			return errors.Wrap(err, "failed to list images")
+		}
+
+		containerList, err := containerStore.List(ctx)
+		if err != nil {
+			return errors.Wrap(err, "failed to list containers")
+		}
+
+		unusedImages := make(map[string]struct{})
+
+		for _, image := range imageList {
+			unusedImages[image.Name] = struct{}{}
+		}
+
+		for _, container := range containerList {
+			delete(unusedImages, container.Image)
+		}
+
+		for image := range unusedImages {
+			opts := []images.DeleteOpt{images.SynchronousDelete()}
+
+			if err := imageStore.Delete(ctx, image, opts...); err != nil {
+				if !errdefs.IsNotFound(err) {
+					if exitErr == nil {
+						exitErr = errors.Wrapf(err, "unable to delete %v", image)
+					}
+					log.G(ctx).WithError(err).Errorf("unable to delete %v", image)
```

Comment:
Does this print to stderr?

Reply:
Should the logging/printing behavior be the same as the removeCommand? I did the same logging format for the same errors from the removeCommand.